src/scripts/main.js

```diff
@@ -5,4 +5,30 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  const createCell = function(data) {
+    const cell = document.createElement('td');
+
+    cell.innerHTML = data;
+
+    row.append(cell);
+  };
```

Comment:
you can create function `createCell` outside of the loop, no need to create it every time

Reply:
and return a cell from function, and use something like 
```
row.append(createCell(person.name));
```